src/test/scala/org/infinispan/spark/suites/StreamingFailOverSuite.scala

```diff
@@ -32,7 +32,7 @@ class StreamingFailOverSuite extends FunSuite with SparkStream with MultipleServ
 
    protected def getProperties = {
       val props = new Properties()
-      props.put("infinispan.client.hotrod.server_list", s"localhost:$getServerPort")
+      props.put("infinispan.client.hotrod.server_list", s"${Cluster.getServerList}")
```

Comment:
I thought the test was about having ```[server0,server1,server2]``` and make sure you could write to Infinispan even when some of the servers were down, since when doing:
```
stream.writeToInfinispan(props)
```

it is currently a discrete operation that means "takes the (non-Infinispan) stream and write to Infinispan following the configuration on ```props```. If props has only 1 server, and this server is down eventually, it will not be able to reach the cluster.